tmt/steps/execute/internal.py

```diff
@@ -228,6 +218,17 @@ def check(self, test):
             except tmt.utils.FileError:
                 return self.check_shell(test)
 
+    def _will_reboot(self, test):
+        """ True if reboot is requested """
+        return os.path.exists(self._reboot_request_paths(test)[0])
+
+    def _reboot_request_paths(self, test):
+        """ Return reboot_request and test_data paths """
+        test_data = os.path.join(
+            self.data_path(test, full=True), tmt.steps.execute.TEST_DATA)
+        reboot_request_path = os.path.join(test_data, REBOOT_REQUEST_FILENAME)
+        return reboot_request_path, test_data
```

Comment:
Hm, returning `test_data` path from `_reboot_request_paths` seems a bit weird to me. I'd keep it to just the `reboot_request_path`... And possibly add another method for `test_data`? The same path seems to be requested also in `execute/__init__.py`. But I feel like that's not very necessary since it's just a one `os.path.join` call.